modules/ossm-extensions-wasmplugin-deploy.adoc

```diff
@@ -0,0 +1,41 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
+////
+:_content-type: PROCEDURE
+[id="ossm-wasmplugin-deploy_{context}"]
+= Deploying `WasmPlugin` resources
+
+You can enable {SMProductName} extensions using the `ServiceMeshExtension` resource. In this example, `istio-system` is the name of the control plane project.  The following example creates an `openid-connect filter` that performs an OpenID Connect flow to authenticate the user.
+
+.Procedure
+
+. Create the following example resource:
++
+.Example ServiceMeshExtension resource extension.yaml
```

Comment:
You can use the filename `plugin.yaml` here and below if you wish. Not a strict requirement though.